fluent-react/src/markup.js

```diff
@@ -1,6 +1,12 @@
-/* eslint-env browser */
-
-const TEMPLATE = document.createElement("template");
+/* global window, require */
+let doc = null;
+if (typeof(window) === "undefined" || !window || !window.document) {
```

Comment:
Could we check for the existence of the `document` global rather than `window` and then `document`?